flake.nix

```diff
@@ -42,7 +41,7 @@
     };
 
     authentik = {
-      url = "github:nix-community/authentik-nix";
+      url = "github:MarcelCoding/authentik-nix";
```

Comment:
they didn't move yet?

Reply:
Sadly, not yet: I've opened a pull request addressing the problems: https://github.com/nix-community/authentik-nix/pull/28